internal/auditlog/init_tinygo.go

```diff
@@ -20,7 +20,7 @@ func init() {
 	})
 
 	// TODO(jcchavezs): check if newest TinyGo supports json.Marshaler for audit log type.
-	RegisterFormatter("json", noopFormater)
-	RegisterFormatter("jsonlegacy", noopFormater)
-	RegisterFormatter("native", nativeFormatter)
+	RegisterFormatter("json", &noopFormatter{})
```

Comment:
Shall we turn this into a variable instead of instantiating this every time? cc @anuraaga

Reply:
Since this is only called once think it's fine